gammapy/maps/tests/test_core.py

```diff
@@ -485,3 +485,21 @@ def test_map_plot_mask():
 
     with mpl_plot_check():
         mask.plot_mask()
+
+
+def test_split_by_axis():
+    time_axis = MapAxis.from_bounds(
+        0,
+        24,
+        nbin=24,
+        unit="hour",
+        name="time",
+    )
+    energy_axis = MapAxis.from_bounds(
+        1, 100, nbin=4, unit="TeV", name="energy", interp="log"
+    )
+    m_4d = Map.create(
+        binsz=0.02, width=(10, 5), frame="galactic", axes=[energy_axis, time_axis]
```

Comment:
I would make it smaller to limit the memory requirements of tests. e.g. use number of pixels instead.